docroot/themes/custom/mass_admin_theme/templates/fields/mass_content_tableau_embed.html.twig

```diff
@@ -12,7 +12,14 @@
  * @see template_preprocess_link_formatter_link_separate()
  */
 #}
-{{ attach_library('mass_theme/tableau') }}
-<div class="ma_tableau_container" style="max-width:100%">
-  <div class="ma_tableau_item" id="{{ randId }}" data-tableau-url="{{ url }}"></div>
-</div>
+{% if token %}
```

Comment:
@arthurbaghdas I know I initially put this check in, but let's pass in the embed type on both of the templates and use that value instead. The problem with token is that if a visualization is switched back to the default style, the token value will still be available even if the field is hidden.